test/shared_test/lib_battery_dispatch_automatic_btm_test.cpp

```diff
@@ -300,7 +300,7 @@ TEST_F(AutoBTMTest_lib_battery_dispatch, TestBasicForecast) {
     batteryPower->connectionMode = ChargeController::AC_CONNECTED;
 
     // Discharge first 4 hours to avoid peak demand charges. Charge while there's solar, then discharge again at 6 pm since this is a high TOU rate
-    std::vector<double> expectedPower = { 50, 50, 50, 34.79, 0, 0, 0, -47.84, -47.84, -47.84, -47.84, -47.84, -47.84, -47.84, -47.84, -47.91, -10.538, 0, 50, 50, 50,
+    std::vector<double> expectedPower = { 50, 50, 50, 37.75, 0, 0, 0, -47.87, -47.97, -47.90, -47.90, -47.90, -47.90, -47.90, -47.90, -47.90, -11.87, 0, 50, 50, 50,
```

Comment:
Increased power due to the higher voltage at low SOC makes total sense! This is great!